src/RoutePatches.tsx

```diff
@@ -76,8 +106,8 @@ export function patchHomePage(serverAPI: ServerAPI, storage: Storage): Mountable
 			"type",
 			(_: Record<string, unknown>[], ret1?: any) => {
 				//console.info('ret1', ret1);
-				updatePlaytimes(storage);
-				ret1.key = Math.random();
+				//updatePlaytimes(storage, true);
+				//ret1.key = Math.random();
 				return ret1;
```

Comment:
So it looks like this patchers no longer needed, because commented code never executed, probably should patchers should be removed as well

Reply:
Same as above, this was going to be a part of cleanup. I was testing multiple things so it was useful to have things to go back to.